### DISCUSSION TITLE
The calculated traffic will be less than the data at the first moment

### DISCUSSION THREAD

**Discussion by MaoKun001013** [pyswmm/pyswmm#484] [Q&A]
Why is it that when I input a time series, the calculated traffic will be less than the data at the first moment? I am using Python 3.9.2 and Pyswmm1.4.0, and what is the reason for this?

**Comment by bemcdonnell** [comment]
@MaoKun001013 can you please provide a figure or some data and code? Some more context would be very helpful in understanding exactly what you’re comparing.
